### ISSUE TITLE
How to keep more than one language? What about commentary streams?

### ISSUE THREAD

**Issue by gbougiakas** [binhex/videoslimmer#11]
Hello, 

thanks for the awesome app. Is it possible to keep more than one languages? If so how?

Second, what is going to happen with commentary streams.

I want to keep English. I have a French movie that contains French (which is fine) and English commentary (if I open it with VLC it says that English is commentary - it is somewhere in the MKV metadata). Will this script leave me with just the English commentary?

**Comment by binhex** [comment]
ok so right NOW it will remove all other languages other than the one you specified, so yes it would leave you only with english commentary in your example. 

the new version that im close to releasing wll allow you to have a comma separated list of languages and that will allow you to keep one or more.

**Comment by gbougiakas** [comment]
OK cheers. If possible have an option to ignore commentary streams? Also is there anyway you could dockerize this? Aren't you the same binhex that makes all these great unraid containers?

**Comment by binhex** [comment]
> If possible have an option to ignore commentary streams?

that might be tricky but i will take a look, depends if they are marked as such in the metadata.



> Aren't you the same binhex that makes all these great unraid containers?

might be :-), yeah i could create a container but tbh this is such a trivial script that it kinda feels wrong to me to run it via docker, i would prefer instead to concentrate on possible integration with usenet/torrent downloader clients.

**Comment by gbougiakas** [comment]
You are right about docker. There is a pull request here by some user who wrote a NZBGet post processing script. I integrated yesterday with nzbget. In the logs of nzbget I see that it worked successfully however, no other log is created (like when I run this manually) to check if it worked and I don't know what's going to happen with radarr trying to pull the downloaded file while it is being processed by the script. I will have to keep an eye on it to figure out how it works.

thanks for all the replies

**Comment by waynetx383** [comment]
For Radarr/Sonarr I created .sh and .bat scripts that take the radarr/sonarr variables and pass them into the command line argument.  I have the script set to run on download and on upgrade. Only downside I've seen is it reports all of the CLI output as errors in the radarr/sonarr logs.
